st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -158,32 +157,50 @@ def run(self, action_parameters):
         if ACTION_OUTPUT_RESULT_DELIMITER in stdout:
             split = stdout.split(ACTION_OUTPUT_RESULT_DELIMITER)
             assert len(split) == 3
-            result = split[1].strip()
+            action_result = split[1].strip()
             stdout = split[0] + split[2]
         else:
-            result = None
-
+            action_result = None
         try:
-            result = json.loads(result)
+            action_result = json.loads(action_result)
         except:
             pass
+        action_status = None
```

Comment:
This could perhaps be refactored in a separate utility method to make it a bit easier to read and test.